lib/msal-core/.eslintrc.json

```diff
@@ -18,7 +18,7 @@
         "@typescript-eslint/ban-types": 0,
         "@typescript-eslint/camelcase": 0,
         "@typescript-eslint/explicit-member-accessibility": 0,
-        "@typescript-eslint/indent": [0, 4, { "SwitchCase": 1 }],
+        "@typescript-eslint/indent": [2, 4, { "SwitchCase": 1 }],
```

Comment:
Does anything change when you enable the normal `indent` rule? I think that's what I did originally that appear to caused a bunch of changes.

Reply:
I think we saw some conflicts there right? so we are defaulting to @ts version?
Theoretically ts lint should have been enforcing that?